istio/istio
# Fix the node agent E2E test, manually tested.

Fixed several things to get it work:
- Update outdated flags value.
- In `start_app.sh`, retain the node agent process log, instead just run it in background.
- Make sure the initial root and cert is loaded.

How to test this:
- `make docker && make docker.push`
- `go test -v  istio.io/istio/security/tests/integration/nodeAgentTest  --tag $(git log -1 --format="%H")  --hub gcr.io/<your-project> -kube-config ~/.kube/config --skip_cleanup`

We should also figure out how to not run `apt-get` install for every test run. Will follow up with key cert generation issue and then ensure it's stability before re-enabling it.

cc @wattli
